av/codec/context.pyx

```diff
@@ -540,6 +540,18 @@ cdef class CodecContext(object):
         def __set__(self, value):
             to_avrational(value, &self.ptr.time_base)
 
+    property codec_tag:
+        def __get__(self):
+            return self.ptr.codec_tag.to_bytes(4, byteorder="little", signed=False).decode(
```

Comment:
I would say stick to ASCII, i don't see anything suggesting this should allow itf8 in a fixed 4 byte budhet

Reply:
Changed to ASCII